content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+### React untuk Designers {#react-for-designers}
 
-If you're coming from a design background, [these resources](https://reactfordesigners.com/) are a great place to get started.
+Jika Anda datang dari latar belakang desain, [sumber-sumber ini](http://reactfordesigners.com/) adalah tempat yang bagus untuk memulai.
 
-### JavaScript Resources {#javascript-resources}
+### Sumber Daya JavaScript {#javascript-resources}
 
-The React documentation assumes some familiarity with programming in the JavaScript language. You don't have to be an expert, but it's harder to learn both React and JavaScript at the same time.
+Dokumentasi React mengasumsikan Anda sudah terbiasa dengan pemrograman menggunakan bahasa JavaScript. Anda tidak harus ahli, tetapi ini akan lebih sulit untuk belajar React dan JavaScript di waktu yang bersamaan.
 
-We recommend going through [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will take you between 30 minutes and an hour but you will feel more confident learning React.
+Kami sarankan untuk menggunakan [ikhtisar JavaScript ini](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) untuk memeriksa tingkatan pengetahuan Anda. Ini akan membutuhkan waktu sekitar 30 menit sampai dengan satu jam tetapi Anda akan merasa lebih percaya diri untuk belajar React.
 
 >Tip
 >
->Whenever you get confused by something in JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](https://javascript.info/) are great websites to check. There are also [community support forums](/community/support.html) where you can ask for help.
+>Ketika Anda dibingungkan oleh sesuatu pada JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) dan [javascript.info](http://javascript.info/) adalah _website_ yang bagus untuk Anda periksa. Ada juga [forum bantuan komunitas](/community/support.html) dimana Anda dapat meminta bantuan.
 
-### Practical Tutorial {#practical-tutorial}
+### Tutorial Praktis {#practical-tutorial}
 
-If you prefer to **learn by doing,** check out our [practical tutorial](/tutorial/tutorial.html). In this tutorial, we build a tic-tac-toe game in React. You might be tempted to skip it because you're not building games -- but give it a chance. The techniques you'll learn in the tutorial are fundamental to building *any* React apps, and mastering it will give you a much deeper understanding.
+Jika Anda lebih suka **belajar dengan mempraktikkan** periksa [tutorial](/tutorial/tutorial.html) kami. Dalam tutorial ini, kami membuat game tic-tac-toe dengan React. Anda mungkin tergoda untuk melewati karena Anda tidak membuat game -- tetapi berikanlah kesempatan untuk mencoba. Teknik yang akan Anda pelajari dalam tutorial ini sangat penting untuk membangun aplikasi *apapun* dengan React, dan menguasainya akan memberi Anda pemahaman yang lebih dalam.
```

Comment:
`learn by doing` should have consistent translation. In here, you translate it as `belajar dengan mempraktikkan` while on my previous comment, you translate it as `belajar sambil melakukan`